src/revpi-functions

```diff
@@ -9,3 +9,12 @@ cm1-detection() {
     grep -m 1 "model name" /proc/cpuinfo | grep -q v6l
     return $?
 }
+
+# Check existance of HAT EEPROM according to document:
+# https://github.com/RevolutionPi/revpi-hat-eeprom/blob/master/RevPi-HAT-EEPROM-Format.md#custom-atoms
```

Comment:
Please use this link: https://github.com/RevolutionPi/revpi-hat-eeprom/blob/master/docs/RevPi-HAT-EEPROM-Format.md#custom-atoms
The spec is moved to the docs directory with the following PR: https://github.com/RevolutionPi/revpi-hat-eeprom/pull/12